src/countItems.js

```diff
@@ -11,12 +11,20 @@
  * @returns {object} An object containing the count of each
  *  string in the input array
  */
-function countItems(array) {
-  // This is your job. :)
+function countItems(array,value) {
```

Comment:
Hmm, what's going on here? You've rewritten two functions to do something different than the prompt.

Is that intentional? That is, do you recognize you're doing something different than asked but still want to demonstrate something? If so, communicate that via a comment in the code or in the PR.

Is it not intentional? That is, do you think you're doing what's being asked? If so, re-read the comment above the function that describes what it is meant to do. It includes:

1. A plain-English description of the function's intended behavior
1. An example input and return value
1. Notes on what _type_ of data the function takes as input and what _type_ of data it returns

Your function takes two arguments (an array and a value) and returns a number. The original takes one argument (an array) and returns an object.

`countItems(['apple', 'goat', 'apple', 'potato'])` should return `{ 'apple': 2, 'goat': 1, 'potato': 1 }`, for example.